### PR TITLE
Plantuml plugin

### PR DESCRIPTION
**Pull request by gattytto**
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
Closes: https://github.com/eclipse/che/issues/17916
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

### CODE REVIEW

**Comment by gattytto** [comment]
tested OK

File: sidecars/plantuml/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+FROM plantuml/plantuml-server
+
+ENV TMPDIR=/tmp
+ENV HOME=/home/theia
+
+ADD etc/storage.conf $HOME/.config/containers/storage.conf
```
Comment (sunix) [inline sidecars/plantuml/Dockerfile:15]: what is that storage.conf for ?

File: che-theia-plugins.yaml [hunk 2]
```diff
@@ -1,5 +1,16 @@
 version: 1.0.0
 plugins:
+  - repository:
```
Comment (sunix) [inline che-theia-plugins.yaml:3]: The plugins part needs to be in a separate PR (the sidecar that would be generated not pushed to quay.io yet) see https://github.com/eclipse/che-plugin-registry/blob/master/CONTRIBUTE.md#contribution-flow

File: sidecars/plantuml/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+FROM plantuml/plantuml-server
+
+ENV TMPDIR=/tmp
```
Comment (sunix) [inline sidecars/plantuml/Dockerfile:12]: Is this mandatory?

**Review by sunix — CHANGES_REQUESTED** [review]

File: che-theia-plugins.yaml (hunk 2 above)
Comment (ericwill) [reply che-theia-plugins.yaml:3]: You can actually merge them together if you really want to, but testing this PR will be difficult because the generated surge link won't contain a valid sidecar reference. :man_shrugging:

**Comment by gattytto** [comment]
closing to refactor with new plungin-registry scheme
